config/locales/exceptions.en.yml

```diff
@@ -139,10 +214,11 @@ en:
         <<: *validation_errors
         code: 'RX157'
         detail: 'Email address is invalid'
-      SM152:
-        <<: *validation_errors
-        code: 'SM152'
-        detail: 'Email address is invalid'
+      MHVAcctCreation155: # Account is already Upgraded
```

Comment:
You're right, I'll double check this and verify with a spec. At some point I think I would like to have a linter for that entire exceptions.en.yml file. I noticed that Rainbows introduced a few, and they're using the standard error `defaults` vs the `external_detaults` for example. A linter for the file would help ensure things don't get too wonky, maybe even help identify duplications.